frontend/src/shared/layout/navigation/navigation.reducer.ts

```diff
@@ -51,10 +58,10 @@ const navigationSlice = createSlice({
                 state.adminItems = undefined;
             }
         },
-        setItemsForProjectName (state, action: PayloadAction<string | undefined>) {
-            const projectName = action.payload;
+        setItemsForProjectName (state, action: PayloadAction<NavigationReducerState | undefined>) {
```

Comment:
Is this action used anywhere else in the code now where it isn't sent a `NavigationReducerState`? If not we can get rid of the `| undefined`. Also assuming we can do that we can remove all the the `action.payload?.enabledServices?.<blah>` with just `action.payload.enabledServices.<blah>` since it will always be valid.

Reply:
Updated to remove optional undefined param.